packages/forms-web-app/src/config.js

```diff
@@ -27,6 +27,10 @@ module.exports = {
     timeout: Number(process.env.DOCUMENTS_SERVICE_API_TIMEOUT || 10000),
     url: process.env.DOCUMENTS_SERVICE_API_URL,
   },
+  pdf: {
+    timeout: Number(process.env.DOCUMENTS_SERVICE_API_TIMEOUT || 10000),
```

Comment:
Maybe duplicate this and rename so it can be controlled separately? 

`PDF_SERVICE_API_TIMEOUT` ?

idk, maybe not needed.